scripts/contracts/src/permission_management/Authorization.sol

```diff
@@ -188,4 +208,15 @@ contract Authorization is IAuthorization, ReservedAddrPublic {
         emit AuthSetted(_account, _permission);
         return true;
     }
+
+    /// @notice Private: delete the permission to the account
+    function _cancelAuth(address _account, address _permission)
+        private
+        returns (bool)
+    {
+        AddressArray.remove(_account, accounts[_permission]);
+        AddressArray.remove(_permission, permissions[_account]);
+        emit AuthCanceled(_account, _permission);
+        return true;
+    }
```

Comment:
https://github.com/cryptape/cita/blob/develop/scripts/contracts/src/permission_management/Authorization.sol#L63, Why not use `_cancelAuth()` directly ?

Reply:
has no right